tests/tests.yaml

```
@@ -47,9 +47,9 @@ answer_tests:
     - yt/frontends/enzo/tests/test_outputs.py:test_ecp
     - yt/frontends/enzo/tests/test_outputs.py:test_nuclei_density_fields
 
-  local_enzo_p_010: # PR 2945
-    - yt/frontends/enzo_p/tests/test_outputs.py:test_hello_world
-    - yt/frontends/enzo_p/tests/test_outputs.py:test_particle_fields
+  local_enzo_e_010: # PR 2945
```

the number should be downgraded
```suggestion
  local_enzo_e_001: # PR 3273
```

---

Oh I see.  So because I'm converting enzo-p to enzo-e, this is effectively the first answer tests associated with this "new" frontend?  OK, that's fine.